.github/workflows/main.yml

```diff
@@ -1,23 +1,23 @@
-# This is a basic workflow to help you get started with Actions
-
 name: CI
 
-# Controls when the action will run. Triggers the workflow on push or pull request
-# events but only for the master branch
 on:
   push:
     branches: [ master ]
   pull_request:
     branches: [ master ]
 
-# A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
-  # This workflow contains a single job called "build"
   build:
-    # The type of runner that the job will run on
     runs-on: ubuntu-latest
+    container: ros:foxy
 
-    # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
-    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
     - uses: actions/checkout@v2
+
+    - run: sudo apt-get update && sudo apt-get install -y curl python3-colcon-coveragepy-result python3-colcon-lcov-result lcov
+
+    - uses: ros-tooling/action-ros-ci@0.0.19
+      with:
+        package-name: simple_planning_simulator
```

Comment:
In order to review, I would need to understand the motivation for the changes. How does this help us? E.g. to add a `simple_planning_simulator`. Is there internal motivation or does this relate to porting to ROS2?

Reply:
@fred-apex-ai this does not add any package, but use it as the build target, we'll have to expand this list with all the packages that need to be built. Right now CI does nothing, it just checks out the source code. In any case, this PR targets master, not the ros2 branch.